src/style/app-fixed.module.css

```
@@ -646,6 +743,69 @@
+  color: var(--createButton-color-active) !important;
+  background-color: var(--createButton-bg-active) !important;
+  border: 1px solid var(--createButton-border-active) !important;
+}
+
+/* Miscellaneous Class used several times */
+
+.modalHeader {
+  border: none;
+  padding-bottom: 0;
+  background-color: var(--primary-bg-color) !important;
+  padding: 1rem 1rem;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-bottom



(lint/suspicious/noShorthandPropertyOverrides)